cmd/serve.go

```diff
@@ -153,15 +153,10 @@ func StartServer(logger *log.Zap, cfg *config.Shield) error {
 	signal.Notify(keystrokeTermChan, os.Interrupt, os.Kill, syscall.SIGTERM)
```

Comment:
It would be better to just create a context that would be cancelled on receiving the signal. See https://github.com/odpf/compass/blob/0ab27e9/cli/serve.go#L55

Reply:
I think this is handled here, CMIIW https://github.com/odpf/shield/blob/44409f5183435b2b8bad8fab2960a866675831a0/cmd/serve.go#L52

Reply:
In that case, can we simplify it by using `signal.NotifyContext(...)`?

Reply:
This is done